transaction.go

```diff
@@ -755,6 +738,25 @@ func (this *Transaction) _BuildTransaction(index int) (*services.Transaction, er
 	if err != nil {
 		return &services.Transaction{}, errors.Wrap(err, "failed to update this ID")
 	}
+
+	// Bellow are checks whether we need to sign the transaction or we already have the same signed
+	if bytes.Equal(signedTx.BodyBytes, updatedBody) {
+		tx := this.signedTransactions._Get(index).(*services.SignedTransaction)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why we get the signed transaction again (I see it's done on line 764 below as well) instead of using `signedTx`?